index.js

```diff
@@ -0,0 +1,46 @@
+const list = {
+  contents: ['机を片付ける', '牛乳を買う', '散歩する'],
+  genres: ['掃除', '買い物', '運動']
+}
+const displayText = `現在持っているタスク一覧\n`;
+
+function listDisplay() {
+  console.log(displayText);
+  for (let i = 0; i < list.contents.length; i++) {
+    console.log(`${i}:[内容]${list.contents[i]}、[ジャンル]${list.genres[i]}`);

```

Comment:
`:`の後ろ`{}`の内側にはスペースを空けるのが通例です。
自動整形ツールを使用するとこういったことが防げます。
https://qiita.com/mitashun/items/e2f118a9ca7b96b97840
```suggestion
    console.log(`${i}: [内容]${ list.contents[i] }、[ジャンル]${list.genres[i] }`);
```